multiqc/utils/search_patterns.yaml

```diff
@@ -495,6 +495,8 @@ samtools/rmdup:
     shared: true
 sargasso:
     fn: 'overall_filtering_summary.txt'
+seqyclean:
+    fn: '*_SummaryStatistics.tsv'
```

Comment:
The [test data](https://github.com/ewels/MultiQC_TestData/tree/master/data/modules/seqyclean) has a mixture of `.tsv` and `.txt` files. Is it always `.tsv` (in which case, we should update the test data repo), or can it be either?

Reply:
Oh wait, sorry - I didn't look at the contents of the file. Never mind 😄 

(For any readers: `.txt` has verbose log output. `.tsv` file has statistics - so we only want the `.tsv´ 👍 )